terraform/10-video-hearing-wowza/modules/wowza/network.tf

```diff
@@ -106,6 +106,18 @@ resource "azurerm_network_security_group" "wowza" {
     destination_address_prefix = var.address_space
   }
 
+  security_rule {
```

Comment:
You do not need this rule.

Reply:
I did not think I did but for certainty and clarity I thought it might be good to have them explicitly on here.
do you see it worth keeping for visibility or just remove?

Reply:
no I don't see any worth in keeping this, I would remove it to avoid any confusion